packages/docs/src/pages/components/upload/examples/ExExpanded.vue

```diff
@@ -30,18 +30,20 @@
   </section>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { defineComponent } from 'vue'
+
+export default defineComponent({
```

Comment:
Please see [this comment](https://github.com/ntohq/buefy-next/pull/269/files#r1732964052).